frontend/class-frontend-page-type.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @package WPSEO\Frontend
+ */
+
+/**
+ * Represents the classifier for determine the current opened page type.
+ */
+class WPSEO_Frontend_Page_Type {
+
+	/**
+	 * @var WPSEO_WooCommerce_Shop_Page
+	 */
+	protected $woocommerce_shop_page;
+
+	/**
+	 * Sets the WooCommmerce shop page object.
```

Comment:
No, the constructor does the setup (construction) for the `WPSEO_Frontend_Page_Type` class. But for constructors, I usually omit the description if nothing interesting happens inside the constructor.